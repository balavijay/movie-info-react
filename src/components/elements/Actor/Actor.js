import React from 'react';
import PropTypes from 'prop-types';

import { StyledActor } from '../../styles/StyledActor';

const Actor = ({ actor }) => (
  <StyledActor>

    <span className="actor-name">{actor.name}</span>

  </StyledActor>
);

Actor.propTypes = {
  actor: PropTypes.object,
}

export default Actor;
