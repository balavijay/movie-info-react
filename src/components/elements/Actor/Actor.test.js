import React from 'react';
import { shallow, mount } from 'enzyme';


import Actor from './Actor';

describe ("Actor Component " , () => {
    let actor = {name: "Test" };
    console.log("actor", actor.name);
   const  app = shallow(<Actor actor={actor} />);
    test("Component render" , () => {
        expect(app).toMatchSnapshot();
    });

    test("Actor component" , () => {

        const wrapper = mount(<Actor actor={actor} />);
        expect(wrapper.find('span').length).toEqual(1);
    });

    

});