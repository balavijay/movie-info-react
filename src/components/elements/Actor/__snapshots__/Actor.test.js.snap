// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actor Component  Component render 1`] = `ShallowWrapper {}`;
